lib/src/tile.dart

```diff
@@ -13,6 +13,19 @@ class Tile {
   int spacing;
   int margin;
 
+  /*
+      Tile Rotation (Clockwise) 
```

Comment:
just curious, did you define this or this a tiled convention?

Reply:
I did define this, though it was easier to work with a single int value. I could change it, then we would translate it in Tiled Component

Reply:
Fine by me to keep it, I was just curious on the origin and also if it was based off something else I would ask to add a reference on the comment.